python/cudf/cudf/_lib/parquet.pyx

```diff
@@ -467,6 +470,10 @@ cdef class ParquetWriter:
     max_page_size_rows: int, default 20000
         Maximum number of rows of each page of the output.
         By default, 20000 will be used.
+    force_nullable_schema : bool, default True.
```

Comment:
We should be consistent in our defaults. Why is `write_parquet` using `False` but `ParquetWriter` using `True`?

This appears to be the only exception -- all other places are using `False` but this class uses `True`.

Reply:
This was intentional. `ParquetWriter` is the chunked Parquet writer, which basically used to differ from the normal Parquet writer(`write_parquet`) in the following way : Chunked Parquet writer always writes every column irrespective of if they really have nulls as `null` in schema. This is done because, we would later want to read all the parquet file chunks and readers expect identical schema. This isn't the case with normal parquet writer, there `not null` and `null` were written to schema based on presence of null values in the column. 

Coming back to the defaults, having `True` in Chunked Parquet writer and `False` in normal Parquet writer as defaults preserve the behavior that has been present until now.

Reply:
Okay. Should we include a comment explaining this in the docs, so readers are aware of this intended difference in behavior?

I'll approve and let you decide how to handle that.

Reply:
Done, documented.